archinstall/lib/menu/menu.py

```diff
@@ -210,12 +210,12 @@ def _show(self) -> MenuSelection:
 			return MenuSelection(type_=MenuSelectionType.Ctrl_c)
 
 		def check_default(elem):
-			if self._default_option is not None and f'{self._default_option} {self._default_str}' in elem:
+			if self._default_option and f'{self._default_option} {self._default_str}' in elem:
 				return self._default_option
 			else:
 				return elem
 
-		if idx is not None:
+		if idx:
```

Comment:
Yepp we need thse `is not None`, because otherwise index of `0` won't work.
![screenshot](https://user-images.githubusercontent.com/861439/168540045-3d8ee550-a161-4dd0-a6cf-ffda61b7e47a.png)

Revert the `None` changes and we're good to go on this PR :)